.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "jenkins/dataproc-custom-images/dataproc-custom-images"]
 	path = jenkins/dataproc-custom-images/dataproc-custom-images
 	url = https://github.com/GoogleCloudPlatform/dataproc-custom-images.git
+[submodule "jenkins/gce-custom-images/cis-harden-images"]
+	path = jenkins/gce-custom-images/cis-harden-images
+	url = https://github.com/broadinstitute/CIS-harden-images.git
```

Comment:
This entry is created (along with a local one in `.git/config`) when we manually run `git submodule add ...` (more info [here](https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/104399223/Callisto+Developer+Handbook#CallistoDeveloperHandbook-AddingaGitRepoasSubmodule)). Something mysterious happened around git auth'ing on the Jenkins node which caused an error and then went away after changing the `https://` url to `git@github`. However,  `.gitmodule` and `.git/config` on the Jenkins node was still showing `https://` for both of the submodules so I'm not convinced that was the issue. Something non-idempotent must be going on. Will keep an eye on it.

Reply:
more modules == more fun :)